x/ibc/core/02-client/keeper/keeper.go

```diff
@@ -117,6 +117,24 @@ func (k Keeper) SetNextClientSequence(ctx sdk.Context, sequence uint64) {
 	store.Set([]byte(types.KeyNextClientSequence), bz)
 }
 
+// GetUpgradePlanHeight gets the IBC upgrade plan height.
+func (k Keeper) GetUpgradePlanHeight(ctx sdk.Context) (uint64, bool) {
+	store := ctx.KVStore(k.storeKey)
```

Comment:
nit: Are we storing this in IBC store? feels weird to store some upgrade stuff in upgrade store and some upgrade stuff in ibc store